bastion/gui/dialogs/firewall_dialog.py

```
@@ -0,0 +1,290 @@
+                    font-weight: bold;
+                }}
+                QPushButton:hover {{
+                    background-color: {color}dd;
+                }}
+            """
+        btn.setStyleSheet(style)
+        btn.setCursor(Qt.CursorShape.PointingHandCursor)
+        return btn
+
+    def start_timer(self):
+        if self.timeout <= 0: return
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)